hipscat/dask_utils.py

```diff
@@ -17,18 +17,23 @@
 from dask.delayed import delayed
 from sklearn.neighbors import KDTree
 
-from . import util
+try:
```

Comment:
this is useful when one-off testing the dask_utils ufuncs without running them in dask parallel. 

So just in the main function, calling the ufunc you are debugging by doing `python dask_utils.py` is a quick way actually see how its breaking instead of the normal traceback that dask throws.